libraries/dagster-iceberg/src/dagster_iceberg/handler.py

```diff
@@ -91,7 +99,33 @@ def handle_output(
             },
         )
 
+    def _get_partition_field_name_prefix(self, context: OutputContext) -> str:
+        """Get partition_field_name_prefix from IO manager config if available, otherwise from asset definition metadata"""
```

Comment:
I think the docstring might be inverted with metadata taking precedence but I may be mistaken.

Reply:
Thanks, good catch!